scripts/timelapse-colorizer-data/generate_data.py

```diff
@@ -241,9 +243,30 @@ def make_dataset(output_dir="./data/", dataset="baby_bear", do_frames=True):
 
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--output_dir", type=str, default="./data/")
-parser.add_argument("--dataset", type=str, default="baby_bear")
-parser.add_argument("--noframes", action="store_true")
+parser.add_argument(
+    "--output_dir",
+    type=str,
+    default="./data/",
+    help="Parent directory to output to. Data will be written to a subdirectory named after the dataset parameter.",
+)
+parser.add_argument(
+    "--dataset",
+    type=str,
+    default="baby_bear",
+    help="Named FMS dataset or FMS id to load.",
```

Comment:
Even though it has nothing to do with scaling, this helpstring should probably say that it has to be a dataset supported by `nuc_morph_analysis.preprocessing.load_data.load_data()`.  We don't support just anything from FMS.